frontend/src/lib/components/EmojiPicker/EmojiPickerPopover.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+    EmojiPicker,
+    EmojiPickerListCategoryHeaderProps,
+    EmojiPickerListEmojiProps,
+    EmojiPickerListRowProps,
+} from 'frimousse'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { Popover } from 'lib/lemon-ui/Popover'
+import { useState } from 'react'
+import { IconEmojiAdd } from '@posthog/icons'
+
+const EmojiPickerCategoryHeader = ({ category, ...props }: EmojiPickerListCategoryHeaderProps): JSX.Element => (
+    <div className="bg-bg-light px-3 pt-3 pb-1.5 font-medium text-neutral-600 text-sm" {...props}>
+        {category.label}
+    </div>
+)
+
+const EmojiPickerEmojiRow = ({ children, ...props }: EmojiPickerListRowProps): JSX.Element => (
+    <div className="scroll-my-1.5 px-1.5" {...props}>
+        {children}
+    </div>
+)
+
+const EmojiPickerEmojiButton = ({ emoji, ...props }: EmojiPickerListEmojiProps): JSX.Element => (
+    <button
+        className="flex items-center justify-center rounded-md text-xl size-8 data-[active]:bg-secondary-3000-hover"
+        {...props}
+    >
+        {emoji.emoji}
+    </button>
+)
```

Comment:
these don't really need to be split out... i was going to add recent emojis in this PR but decided to split it
i'll need these as separate components in that follow-up since I'll be adding a custom row of results